example/index.jsx

```diff
@@ -5,7 +5,7 @@ import logo from './logo.svg';
 import AnimatedModal from './components/animatedModal';
 import SwipeInFromBelowModal from './components/SwipeInFromBelowModal';
 import WobblyModal from './components/wobblyModal';
-import { ModalPortal, Modal } from '@weahead/react-customizable-modal';
+import { ModalPortal, Modal } from '../';
```

Comment:
@jb-san Do we want this change in the example? Shouldn't it be copy/paste-able? These two changes basically tie it to the repo instead of being portable?

Reply:
no, its not suposed to be copy pasteable, this should be to test the components and hooks when developing, maybe changing the name to something to be more clear ?